horace_core/algorithms/fake_sqw.m

```diff
@@ -190,7 +195,8 @@
         pix_range_l = run_files{i}.calc_pix_range(en_lo(i),en_hi(i),cache_opt{:});
         img_range = [min(pix_range_l(1,:),img_range(1,:));max(pix_range_l(2,:),img_range(2,:))];
     end
-    img_range=range_add_border(img_range,1e-6);     % add a border to account for Matlab matrix multiplication bug
+    img_range=range_add_border(img_range,...
+        data_sqw_dnd.border_size);     % add a border to account for Matlab matrix multiplication bug
```

Comment:
What is the "Matlab matrix multiplication bug"

Reply:
have no idea (Toby's writing) but I suspect just round-off errors, affecting the allocation of the pixels at boundary.  Surprisingly big round-off error as eps-size border gives ~10^-6 abs-err in transformed pix range (ill conditioned matrix somewhere?)